demo/src/__tests__/App.spec.tsx

```diff
@@ -9,7 +9,7 @@ describe("App", () => {
     });
 
     it("renders Output heading", () => {
-        expect(screen.getByText("Output").nodeName).toEqual("H2");
+        expect(screen.getByText("Output").nodeName).toEqual("H3");
```

Comment:
It's not good to change heading levels just to change style, as the heading levels should not increase by more than one at a time. Keep as `h2`.